configuration-maven-extension/src/main/java/com/gradle/jenkins/maven/extension/internal/GradleEnterpriseConfigurerListener.java

```diff
@@ -0,0 +1,45 @@
+package com.gradle.jenkins.maven.extension.internal;
+
+import com.gradle.maven.extension.api.GradleEnterpriseApi;
+import com.gradle.maven.extension.api.GradleEnterpriseListener;
+import org.apache.maven.execution.MavenSession;
+import org.codehaus.plexus.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(
+    role = GradleEnterpriseListener.class,
+    hint = "gradle-enterprise-configurer"
+)
+public class GradleEnterpriseConfigurerListener implements GradleEnterpriseListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GradleEnterpriseConfigurerListener.class);
+
+    // Have to be in sync with what is set in MavenBuildScanInjection
```

Comment:
why can't we put them in a shared place ?

Reply:
We definitely can. But this is the only thing that could be shared so far. At this moment I'm not sure that 2 constants can justify the module creation.